### DISCUSSION TITLE
Linking to symbols in other files does not like paths with a hash #

### DISCUSSION THREAD

**Discussion by danspurway-tombola** [community/community#60497] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

In terms of this feature: https://docs.github.com/en/repositories/working-with-files/using-files/navigating-code-on-github#jumping-to-the-definition-of-a-function-or-method

I have found that if a file path includes a hash (#), that the navigated path does not escape the # character

![image](https://github.com/community/community/assets/40992349/43959a0a-3bf3-4f58-b546-fb504f74ee5d)

URL that is navigated to: https://github.com/ORG/REPO/blob/master/Scripts/New/C#/Lib/FeatureToggles.cs#L60
![image](https://github.com/community/community/assets/40992349/0bb02685-2481-4e52-9e0d-571e9a967bad)

URL that works: https://github.com/ORG/REPO/blob/master/Scripts/New/C%23/Lib/FeatureToggles.cs#L60

**Comment by Akash1134** [comment]
Hi @danspurway-tombola ,

Thank you for sharing the feedback. We have shared it internally with the team. 🙂
